packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
+    getErrorPrefix: () => string;
+    historical: Historical;
+}) {
+    return function useSubscriptionHook(query: ToriiQueryBuilder<Schema>) {
+        // Subscription handle to update
+        const subscriptionRef = useRef<Subscription | null>(null);
+        // Handle to user input query
+        const fetchingRef = useRef<ToriiQueryBuilder<Schema> | null>(null);
+        // Async lock to sync with event loop
+        const isUpdating = useRef<boolean>(false);
+
+        const fetchData = useCallback(async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: config.getErrorPrefix

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)